.github/workflows/main.yml

```diff
@@ -12,13 +12,28 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
needed to add the "ref" otherwise lighthouse wouldn't appear in the status checks. Seems to work ok for pull request but not sure how it affects main branch builds. (we'll find out after merging) https://github.com/GoogleChrome/lighthouse-ci/issues/172#issuecomment-644127446